samples/csharp_dotnetcore/56.teams-file-upload/README.md

```diff
@@ -52,9 +52,11 @@ the Teams service needs to call into the bot.
 
 > Note this `manifest.json` specified that the bot will be installed in "personal" scope which is why you immediately entered a one on one chat conversation with the bot. Please refer to Teams documentation for more details.
 
-Sending a message to the bot will cause it to respond with a card that will prompt you to upload a file. The file that's being uploaded is the `teams-logo.png` in the `Files` directory in this sample. The `Accept` and `Decline` events illustrated in this sample are specific to Teams. You can message the bot again to receive another prompt.
+1. Sending a message to the bot will cause it to respond with a card that will prompt you to upload a file. The file that's being uploaded is the `teams-logo.png` in the `Files` directory in this sample. The `Accept` and `Decline` events illustrated in this sample are specific to Teams. You can message the bot again to receive another prompt.
 
-You can also send a file to the bot as an attachment in the message compose section in Teams. This will be delivered to the bot as a Message Activity and the code in this sample fetches and saves the file.
+1. You can send a file to the bot as an attachment in the message compose section in Teams. This will be delivered to the bot as a Message Activity and the code in this sample fetches and saves the file.
+
+1. You can also send inline image in message compose section. This will part of attachments in Activity and requires Bot's access token to fetch the image.
```

Comment:
Sorry, i do not understand what this means?

`You can also send inline image in message compose section. This will part of attachments in Activity and requires Bot's access token to fetch the image.`

Reply:
@EricDahlvang  - For image files we have two options to upload:
1. Upload image as attachment: 
![image](https://user-images.githubusercontent.com/31851992/104083098-ba8c6300-5261-11eb-84c1-687beca39184.png)
This is currently handled in existing sample here: [TeamsFileUploadBot.cs#L36](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/56.teams-file-upload/Bots/TeamsFileUploadBot.cs#L36) 
1. Copy paste Inline image:
![image](https://user-images.githubusercontent.com/31851992/104083128-10f9a180-5262-11eb-8ee4-f15b0aa7c520.png)
This PR shows how to fetch these inline images. [TeamsFileUploadBot.cs#L58](https://github.com/Wajeed-msft/BotBuilder-Samples/blob/washaik/InlineImageUpload/samples/csharp_dotnetcore/56.teams-file-upload/Bots/TeamsFileUploadBot.cs#L58)